Disher/Models/Recipe.m

```diff
@@ -27,4 +27,9 @@ + (Recipe *) initWithRecipe:(NSString *) name withURL:(NSString *) imgURL withSo
     return newRecipe;
 }
 
++ (Recipe *) getRecipeWithID:(NSString *)parseRecipeID {
+    PFQuery *query = [PFQuery queryWithClassName:@"Recipe"];
+    Recipe *returnedRecipe = [query getObjectWithId:parseRecipeID];
+    return returnedRecipe;
```

Comment:
is it possible just directly return it?  
`return [query getObjectWithId:parseRecipeID];`